src/components/InpageNav/InpageNav.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+type InpageNavItemLinksProps = {
+  links?: InpageNavItemProps[];
+};
+
+type InpageNavItemProps = {
+  title?: string;
+  id?: string;
+};
+
+export const InpageNav = ({ links }: InpageNavItemLinksProps) => {
+  return (
+    <>
+      <h3 className="section-links__heading">On this page</h3>
```

Comment:
I think a `<strong>` tag would prob be more appropriate for this than `<h3>`. @chriddell your thoughts?

Reply:
Do you think this will require class as well.` <strong class"">`

Reply:
It will need a class yes (`className` in React)  and that should stay the same. The strong tag simply implies more importance than standard text and although that would usually mean it was styled bold that should have nothing to do with how it actually looks